### PR TITLE
[19.03] bump ver 19.03.0

### PR DESCRIPTION
**Pull request by andrewhsu**
bump ver 19.03.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 For official release notes for Docker Engine CE and Docker Engine EE, visit the
 [release notes page](https://docs.docker.com/engine/release-notes/).
 
-## 19.03.0 (2019-06-DD)
+## 19.03.0 (2019-06-17)
```
Comment (tonistiigi) [inline CHANGELOG.md]: Fixes:

- build: add SSH agent socket forwarder is not new in 19.03 
- builder-next: update buildkit to v0.5.0 <- not the last version
- cache-from <- put to a separate line

New things:
- BuildKit based builds use Dockerfile frontend version `docker/dockerfile:1.1` by default
- BuildKit based builds do not rely on an external image anymore for COPY/ADD operations
- BuildKit based builds will first look for a Dockerfile specific dockerignore file (eg. Dockerfile.dockerignore) for ignored paths
- BuildKit based builds automatically detect if process execution is possible for x86, arm and arm64 binaries.

Comment (andrewhsu) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,69 +3,80 @@
...
+### Deprecation
+
+* Remove v1 manifest support, remove `--disable-legacy-registry`. Pushing v1 manifests to registries is no longer possible, pushing schema v2 (or OCI) manifests is now the only way. However, pulling v1 manifests is still possible. [moby/moby#37874](https://github.com/moby/moby/pull/37874)
+* Remove v1.10 migrator. [moby/moby#38265](https://github.com/moby/moby/pull/38265)
+* Skip deprecated storage-drivers in auto-selection. [moby/moby#38019](https://github.com/moby/moby/pull/38019)
+* Deprecate AuFS storage driver, and add warning. [moby/moby#38090](https://github.com/moby/moby/pull/38090)
 
 ### Client
 
++ build: allow setting buildkit outputs. [docker/cli#1766](https://github.com/docker/cli/pull/1766)
```
Comment (thaJeztah) [inline CHANGELOG.md]: should this one be under "builder"?

```suggestion
+ builder-next: allow setting buildkit outputs. [docker/cli#1766](https://github.com/docker/cli/pull/1766)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -77,43 +88,43 @@ For official release notes for Docker Engine CE and Docker Engine EE, visit the
 
 ### Runtime
 
-* Allow running dockerd as a non-root user (Rootless mode). [moby/moby#380050](https://github.com/moby/moby/pull/38050)
-* Rootless: optional support for `lxc-user-nic` SUID binary. [docker/engine#208](https://github.com/docker/engine/pull/208)
-* Add DeviceRequests to HostConfig to support NVIDIA GPUs. [moby/moby#38828](https://github.com/moby/moby/pull/38828)
++ Allow running dockerd as a non-root user (Rootless mode). [moby/moby#380050](https://github.com/moby/moby/pull/38050)
++ Rootless: optional support for `lxc-user-nic` SUID binary. [docker/engine#208](https://github.com/docker/engine/pull/208)
++ Add DeviceRequests to HostConfig to support NVIDIA GPUs. [moby/moby#38828](https://github.com/moby/moby/pull/38828)
++ Add --device support for Windows. [moby/moby#37638](https://github.com/moby/moby/pull/37638)
```
Comment (thaJeztah) [inline CHANGELOG.md]:
```suggestion
+ Add `--device` support for Windows. [moby/moby#37638](https://github.com/moby/moby/pull/37638)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -77,43 +88,43 @@ For official release notes for Docker Engine CE and Docker Engine EE, visit the
...
 * Bump Golang to 1.12.5. [docker/engine#209](https://github.com/docker/engine/pull/209)
 * Bump containerd to 1.2.6. [moby/moby#39016](https://github.com/moby/moby/pull/39016)
 * Bump runc to 1.0.0-rc8, opencontainers/selinux v1.2.2. [docker/engine#210](https://github.com/docker/engine/pull/210)
 * Bump google.golang.org/grpc to v1.20.1. [docker/engine#215](https://github.com/docker/engine/pull/215)
-* Fix: fetch the right device number when greater than 255 and using `--device-read-bps` option. [moby/moby#39212](https://github.com/moby/moby/pull/39212)
-* Fix: "Path does not exist" error when path definitely exists. [moby/moby#39251](https://github.com/moby/moby/pull/39251)
 * Performance optimizations in aufs and layer store for massively parallel container creation/removal. [moby/moby#39135](https://github.com/moby/moby/pull/39135) [moby/moby#39209](https://github.com/moby/moby/pull/39209)
 * Pass root to chroot to for chroot Tar/Untar (CVE-2018-15664) [moby/moby#39292](https://github.com/moby/moby/pull/39292)
+- Fix docker --init with /dev bind mount. [moby/moby#37665](https://github.com/moby/moby/pull/37665)
```
Comment (thaJeztah) [inline CHANGELOG.md]:
```suggestion
- Fix docker `--init` with `/dev` bind mount. [moby/moby#37665](https://github.com/moby/moby/pull/37665)
```

File: CHANGELOG.md (hunk 2 above)
Comment (andrewhsu) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (andrewhsu) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (andrewhsu) [reply CHANGELOG.md]: done

**Pull request merged**